### PR TITLE
118 task show menu location in the title box

### PR DESCRIPTION
**Pull request by Tearran**
# Description

This change updates the title of the dialog menu to include the `parent_id`, enhancing clarity for users by indicating the context of the current menu. This change addresses issue #118 by providing a clearer menu location reference within the title box.

**Motivation and Context:**  
The addition of the `parent_id` in the title allows users to easily identify the hierarchy of menus, improving usability and reducing confusion when navigating through multiple layers of options.

**Issue Reference:**  
- Issue #118

# Implementation Details

- **Key Changes Introduced by this PR:**  
  - Modified the title in the dialog menu from `"$TITLE "` to `"$TITLE ($parent_id)"`, which now displays the parent menu's identifier.

# examples

# Testing Procedure

- **Test 1:**  
  - **Description:** Verified the dialog menu title displays correctly with the `parent_id` included.  
  - **Results:** The title displayed as expected, confirming the inclusion of the `parent_id`.

- **Test 2:**  
  - **Description:** Navigated through multiple menu layers to ensure that the titles updated correctly in all instances.  
  - **Results:** Titles in all menus showed the correct context, with no errors encountered during navigation.

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have ensured that my changes do not introduce new warnings or errors
- [x] No new external dependencies are included
- [x] Changes have been tested and verified
- [x] I have included necessary metadata in the code, including associative arrays

### CODE REVIEW

**Comment by Tearran** [comment]
image

image

**Comment by igorpecovnik** [comment]
If you go one menu lower, it should retain top section. SSH is example of this. Also i would propose to remove this status from BACKTITLE. Once is IMO enough. Lets keep this status place clean and ready for something else.

**Comment by Tearran** [comment]
>If you go one menu lower, it should retain top section. SSH is example of this. 


upon further review of whiptail. The use the "--menu" "header" option is typically used as a prompt something like "Select and option:"

we can add "Breadcrumbs" here

Note: Changed sub menu name from S07.1 to SSH  
Note: --backtitle at the top 
image

Note: --title uses runtime valuable $TITLE  

image

File: lib/armbian-configng/config.ng.functions.sh [hunk 1]
```diff
@@ -228,7 +228,7 @@ function set_runtime_variables() {
 	[[ -z "${ARMBIAN// /}" ]] && ARMBIAN="$DISTRO $DISTROID"
 	DEFAULT_ADAPTER=$(ip -4 route ls | grep default | tail -1 | grep -Po '(?<=dev )(\S+)')
 	LOCALIPADD=$(ip -4 addr show dev $DEFAULT_ADAPTER | awk '/inet/ {print $2}' | cut -d'/' -f1)
-	BACKTITLE="\n    Contribute: https://github.com/armbian/configng\n"
```
Comment (Tearran) [inline lib/armbian-configng/config.ng.functions.sh:231]: lightweight limitation of whiptail --backtitle does not seem to support \n

File: lib/armbian-configng/config.ng.jobs.json [hunk 2]
```diff
@@ -79,7 +79,7 @@
                     "condition": "[[ -n $(ls /sbin/armbian-install) ]]"
                 },
                 {
-                    "id": "S07.1",
+                    "id": "SSH",
                     "description": "Manage SSH login options",
```
Comment (Tearran) [inline lib/armbian-configng/config.ng.jobs.json:83]: ID limitation can not be used elsewhere.

**Comment by Tearran** [comment]
## Top/Main menu
--backtitle = "Canned placeholder"
--title = "Canned placeholer"
--menu =  Menu "header/prompt" basic runtime information of system

image

>If you go one menu lower, it should retain top section. 

--menu = Left blank placeholder "$status"
Breadcrumbs placed in title

image

>If you go one menu lower, it should retain top section.
 
2nd Breadcrumb in title

image

**Comment by Tearran** [comment]
## Top/Main menu function

independent function
--backtitle = "$BACKTITLE" from set_runtime_variables
--title = "$TITLE" from set_runtime_variables
--menu =  Menu "header/prompt" basic runtime from set_runtime_variables

image

## Sub menu loop function

> Also i would propose to remove this status from BACKTITLE. Once is IMO enough. Lets keep this status place clean and ready for something else.

--menu  blank placeholder "$status" to be extended or replaced

Breadcrumbs placed in title

image

>If you go one menu lower, it should retain top section.
 
2nd Breadcrumb in --title

image

**Review by igorpecovnik — APPROVED** [review]

**Pull request merged**